app/models/photo.rb

```diff
@@ -0,0 +1,10 @@
+class Photo < ActiveRecord::Base
+  has_attached_file :image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [120/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)